src/components/CommunityAnnotationTable/components/VotesTableCell.tsx

```diff
@@ -0,0 +1,167 @@
+import React from 'react';
+import * as R from 'ramda';
+import { BackendAPI } from '../../common/API';
+import { Icon, Button, Popup } from 'semantic-ui-react';
+import { withRouter, RouteComponentProps } from 'react-router-dom';
+import { withCookies, ReactCookieProps, Cookies } from 'react-cookie';
+
+import {
+    CommunityAnnotationData,
+    CommunityVotes,
+    makeTableColumnData,
+} from '../model';
+
+type VotesTableCellProps = {
+    value: {
+        data: CommunityAnnotationData;
+    } & CommunityVotes;
+    activeFeature: any;
+} & RouteComponentProps<{ uuid: string }>;
+
+type VotesTableCellState = {
+    status: string;
+};
+
+class VotesTableCell extends React.Component<
+    VotesTableCellProps & ReactCookieProps,
+    VotesTableCellState
+> {
+    constructor(props: VotesTableCellProps & ReactCookieProps) {
+        super(props);
+        this.state = {
+            status: 'ready',
+        };
+    }
+
+    static getORCIDData(cookies: Cookies) {
```

Comment:
Can this be a separate function? Maybe in the model?

Reply:
yes I'll move it to `model.ts`